frontend/package.json

```diff
@@ -95,9 +95,9 @@
     "ci-test": "CI=true jest --env=jsdom --testResultsProcessor=jest-sonar-reporter --updateSnapshot --runInBand --coverage",
     "lint": "eslint",
     "format:check": "prettier --config .prettierrc.json",
-    "format:check-all": "prettier --config .prettierrc.json src/**/*.js",
+    "format:check-all": "prettier --config .prettierrc.json src/*.js && prettier --config .prettierrc.json src/**/*.js && prettier --config .prettierrc.json src/**/**/*.js",
     "format:write": "prettier --config .prettierrc.json --write",
-    "format:write-all": "prettier --config .prettierrc.json src/**/*.js --write",
+    "format:write-all": "prettier --config .prettierrc.json src/*.js --write && prettier --config .prettierrc.json src/**/*.js --write && prettier --config .prettierrc.json src/**/**/*.js --write",
```

Comment:
@FreshGyan do you have any idea why this would be necessary to hit the different directory levels? I don't think I've ever needed to do this before. I added it like this to get us unblocked for now